nais.yaml

```diff
@@ -24,12 +24,7 @@ spec:
   replicas:
     min: 1
     max: 4
-  ingresses: [
-    "https://sosialhjelp-innsyn.dev-sbs.nais.io/sosialhjelp/innsyn",
-    "https://www-q1.nav.no/soknadsosialhjelp/innsyn",
-    "https://www-q0.nav.no/sosialhjelp/innsyn",
-    "https://www-q1.nav.no/sosialhjelp/innsyn",
-  ]
+  ingresses: {{{ingresses}}}
```

Comment:
Skal det være {{{ tre slike?

Reply:
Nei, det er copy+pasta fra sosialhjelp-innsyn-api. Jeg tenkte det var Go template-syntaks for lister, men det fungerer fint med bare to. :+1: